volumina/navigationController.py

```diff
@@ -33,7 +33,7 @@
 
 def posView2D(pos3d, axis):
     """convert from a 3D position to a 2D position on the slicing plane
-       perpendicular to axis"""
+    perpendicular to axis"""
```

Comment:
if our hooks do it then it's fine ;). I guess on our side a better way to handle this would be pinning black in the pre-commit hook to some specific version (black is still evolving...).